app/services/search/tag.rb

```diff
@@ -5,33 +5,31 @@ class Tag < Base
     MAPPINGS = JSON.parse(File.read("config/elasticsearch/mappings/tags.json"), symbolize_names: true).freeze
 
     class << self
-      def search(query_string)
-        request do
-          SearchClient.search(
-            index: INDEX_ALIAS,
-            body: {
-              query: {
-                query_string: {
-                  query: query_string,
-                  analyze_wildcard: true,
-                  allow_leading_wildcard: false
-                }
-              },
-              sort: {
-                hotness_score: "desc"
-              }
-            },
-          )
-        end
-      end
-
       def search_documents(query_string)
         results = search(query_string)
         results.dig("hits", "hits").map { |tag_doc| tag_doc.dig("_source") }
       end
 
       private
 
+      def search(query_string)
```

Comment:
this should be private, as it is in `Search::ChatChannelMembership`

Reply:
💯 I think that should be done when we give these search methods a good refactor and move `search` into the base class.